deps/Versions.make

```diff
@@ -1,4 +1,4 @@
-LLVM_VER = 3.7.1
+LLVM_VER = 3.8.0
```

Comment:
we'll probably do 3.8.1 some time not too long after branching for 0.6-dev, but it'll need CI preparation